packages/devnet/contracts/Devnet.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: Apache-2.0
+/// @title Devnet
+pragma solidity ^0.8.13;
+
+import {IConsensus} from "@cartesi/rollups/contracts/consensus/IConsensus.sol";
+import {Authority} from "@cartesi/rollups/contracts/consensus/authority/Authority.sol";
+import {History} from "@cartesi/rollups/contracts/history/History.sol";
+import {IAuthorityHistoryPairFactory} from "@cartesi/rollups/contracts/consensus/authority/IAuthorityHistoryPairFactory.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {IMarketplace} from "@sunodo/contracts/contracts/marketplace/IMarketplace.sol";
+import {IValidatorNodeProvider} from "@sunodo/contracts/contracts/provider/INodeProvider.sol";
+
+contract Devnet {
+    IERC20 public immutable token;
```

Comment:
Is the `token()` function ever called, once the `Devnet` contract is deployed?
If not, we can remove this state variable and the assignment at line 24.

Reply:
Removed